### PR TITLE
chore(www): update discord_icon

### PR DESCRIPTION
**Pull request by Usamaliaquat123**
#21661

### CODE REVIEW

**Comment by tesseralis** [comment]
It seems like the build is failing:

```
03:23:43 AM: error Generating JavaScript bundles failed /usr/src/app/www/www/node_modules/react-icons/fa/index.js: Support for the experimental syntax 'exportDefaultFrom' isn't currently enabled (1:8): > 1 | export Fa500px from './500px'; | ^ 2 | export FaAdjust from './adjust'; 3 | export FaAdn from './adn'; 4 | export FaAlignCenter from './align-center'; Add @babel/plugin-proposal-export-default-from (https://git.io/vb4yH) to the 'plugins' section of your Babel config to enable transformation.
```

File: www/src/components/navigation.js [hunk 1]
```diff
@@ -3,10 +3,9 @@ import { jsx } from "theme-ui"
 import { useColorMode } from "theme-ui"
 import GithubIcon from "react-icons/lib/go/mark-github"
 import TwitterIcon from "react-icons/lib/fa/twitter"
-
 import Link from "../components/localized-link"
 import SearchForm from "./search-form"
-import DiscordIcon from "../components/discord"
+import {FaDiscord}  from "react-icons/fa";
```
Comment (tesseralis) [inline www/src/components/navigation.js]: Could you do

```
import DiscordIcon from "react-icons/fa/discord"
```

This is what we're doing with the other icons.

Comment (Usamaliaquat123) [reply www/src/components/navigation.js]: sure

**Comment by mohouyizme** [comment]
FontAwesome icons in `react-icons@2` don't contain a discord icon.

**Comment by mohouyizme** [comment]
@tesseralis what do you think about upgrading to `react-icons@3.x.x`?

**Comment by tesseralis** [comment]
@mohouyizme @usamaliaquat123 we should probably upgrade to react-icons 3 :P

**Comment by akshitkrnagpal** [comment]
I think upgrading should be a separate PR since we will need to fix import paths and check new bundle size in case not used icons are being ended up in a bundle. https://github.com/react-icons/react-icons#migrating-from-version-2---3

Should I create a new issue for that?

**Comment by akshitkrnagpal** [comment]
Anyone here wants to tackle it? https://github.com/gatsbyjs/gatsby/issues/21859

**Comment by pieh** [comment]
I'll close this PR in favour of https://github.com/gatsbyjs/gatsby/pull/21860 which is handling entire react-icons migration from 2 to 3

**Pull request closed without merging**